compiler/packages/babel-plugin-react-compiler/src/ReactiveScopes/CodegenReactiveFunction.ts

```diff
@@ -2665,3 +2667,38 @@ function compareScopeDeclaration(
   else if (aName > bName) return 1;
   else return 0;
 }
+
+function setMissingLocationsToNull(ast: any): void {
+  if (Array.isArray(ast)) {
+    ast.forEach(item => setMissingLocationsToNull(item));
+    return;
+  } else if (
+    ast == null ||
+    typeof ast !== 'object' ||
+    typeof ast['type'] !== 'string'
+  ) {
+    return;
+  }
+  if (ast['loc'] == null) {
+    ast['loc'] = {
+      start: {line: null, column: null, index: null},
+      end: {line: null, column: null, index: null},
+      filename: null,
+      identifierName: null,
+    };
```

Comment:
At least on the version of Babel we're using for tests, just setting `loc: null` wasn't enough to exclude the node. But setting all of this was enough to exclude them.